### PR TITLE
Upgrade dependencies to the latest beta

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-buttons bundle size difference from 5.16.7 to ae3c920aa3bd7c7ac49abfd0ef536da48d8ef11a
css, master: 20.1kb decrease (-0.30kb/gzip)
css, internal: 24.33kb decrease (-0.33kb/gzip)
css, whitelabel: 87.51kb decrease (-2.19kb/gzip)

File: demos/src/custom-icon-button.mustache [hunk 1]
```diff
@@ -12,11 +12,12 @@
 	</span>
 </button>
 
-<button class="o-buttons o-buttons--big o-buttons--primary o-buttons--custom-claret o-buttons-icon o-buttons-icon--book">
+
+<button class="o-buttons o-buttons--secondary o-buttons--big o-buttons--custom-claret o-buttons-icon o-buttons-icon--book">
 	Book!
 </button>
 
-<button class="o-buttons o-buttons--secondary o-buttons--big o-buttons--custom-claret o-buttons-icon o-buttons-icon--book">
```
Comment (notlee) [inline demos/src/custom-icon-button.mustache]: This is unrelated -- reorders buttons in the demo to look better
